source/MaterialXGenShader/Util.cpp

```diff
@@ -114,7 +114,7 @@ void getFilesInDirectory(const std::string& directory, StringVec& files, const s
 
 bool readFile(const string& filename, string& contents)
 {
-    std::ifstream file(filename, std::ios::in | std::ios::binary);
+    std::ifstream file(filename, std::ios::in);
```

Comment:
Changing to text mode as it handles different line ending conventions automatically.